libs/barista-components/chart/src/heatfield/chart-heatfield.html

```diff
@@ -1,13 +1,16 @@
 <button
   class="dt-chart-heatfield-marker"
+  [class.dt-chart-heatfield-expanded]="active || alwaysExpanded"
   [class.dt-chart-heatfield-active]="active"
   [style.visibility]="_isValidStartEndRange ? 'visible' : 'hidden'"
   #marker="cdkOverlayOrigin"
   cdk-overlay-origin
   (click)="_toggleActive()"
   [attr.aria-label]="ariaLabel"
   [attr.aria-labelledby]="ariaLabelledBy"
-></button>
+>
+  <span class="dt-chart-heatfield-text">{{ text }}</span>
```

Comment:
This span is always rendered, even if there is not text in there, should we make this conditional?